tsconfig.json

```diff
@@ -9,6 +9,6 @@
 		"skipLibCheck": true,
 		"sourceMap": true,
 		"strict": true,
-		"moduleResolution": "node16"
+		"module": "preserve",
```

Comment:
Why this change?

Reply:
This package cannot be used with SSR, and I don't think setting nodeResolution to node16 is necessary.

I can revert this.